autopilot/contractor.go

```diff
@@ -764,38 +781,37 @@ func contractMapBool(contracts []types.FileContractID) map[types.FileContractID]
 	return cmap
 }
 
-func calculateHostCollateral(cfg api.AutopilotConfig, settings rhpv2.HostSettings, renterFunds, txnFee types.Currency) (types.Currency, error) {
-	// check underflow
-	if settings.ContractPrice.Add(txnFee).Cmp(renterFunds) > 0 {
-		return types.ZeroCurrency, errors.New("contract price + fees exceeds funding")
+func renewContractCollateral(settings rhpv2.HostSettings, renterFunds types.Currency) types.Currency {
```

Comment:
resolving this because I think it's outdated, will update the PR to provide the collateral to the `worker` as mentioned here https://github.com/SiaFoundation/renterd/pull/104#discussion_r1085831248